### ISSUE TITLE
push to apple app store

### ISSUE THREAD

**Issue by jessfraz** [KittyCAD/modeling-app#581]
okay so we got kinda far... we can publish to test flight etc.

the biggest problem is getting the perms we need to read from the filesystem. See: https://developer.apple.com/documentation/security/app_sandbox/migrating_your_app_s_files_to_its_app_sandbox_container

- [ ] Migrate to `~/Library/Containers/{our bundle name}`

You have to have your app in a sandbox container in order to be accepted which is fine. Right now we use `~/Library/Application Support/{our bundle name}` for where we store global settings we need to move that to `~/Library/Containers/{our bundle name}` so that part is the easiest part of this.

- [ ] Prompt user for access to any files where we get denied

This is the harder part. We kinda assume we just have access to any thing in ~/Documents and the user's home folder. So basically whenever we try to access a specific file or directory and we get denied we need to prompt the user for access.

**Comment by franknoirot** [comment]
I don't think we publish to the app store yet, so I think we still need to move that data folder and figure out the permissions requesting. Surely there is an Electron API for that
